app/views/state_file/questions/w2/edit.html.erb

```diff
@@ -26,7 +26,7 @@
               <%= f.vita_min_money_field(
                     field_name.to_sym,
                     t(".box14_#{code['name'].downcase}_html"),
-                    options: { value: value || 0 },
+                    options: { value: value },
```

Comment:
removed b/c we were pre-filling with 0 and then showing them an error when they saved to "put in 0"